siteapp/views.py

```diff
@@ -1438,4 +1438,19 @@ def organization_settings_save(request):
         users = users[:20] # limit
         return JsonResponse({ "users": [user.render_context_dict(request.organization) for user in users] })
 
-    return JsonResponse({ "status": "error", "message": str(request.POST) })
\ No newline at end of file
+    return JsonResponse({ "status": "error", "message": str(request.POST) })
+
+def shared_static_pages(request, page):
+    from django.utils.module_loading import import_string
+    from django.contrib.humanize.templatetags.humanize import intcomma
+    password_hasher = import_string(settings.PASSWORD_HASHERS[0])()
+    password_hash_method = password_hasher.algorithm.upper().replace("_", " ") \
+        + " (" + intcomma(password_hasher.iterations) + " iterations)"
+
```

Comment:
Can you explain more about the password hasher?

Reply:
The privacy policy we're adopting has a section that says what password hashing method is actually in use on the site. This fetches that information at run time rather than hard-coding it into the privacy policy document itself and sends the description of the hashing method to the template.

Reply:
Thanks